README.md

```diff
@@ -360,6 +360,39 @@ You'll be redirected automatically to an Okta login page. Once you successfully
 
 This module integrates with Spring Security's OAuth support, all you need is the mark your application with the standard `@EnableOAuth2Client` annotation. 
 
+## Use with Spring Native
+
+You can use this starter with [Spring Native](https://github.com/spring-projects-experimental/spring-native). However, you will need to add some annotations to your main Spring Boot application class.
+
+```java
+package com.example.demo;
+
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.nativex.hint.AccessBits;
+import org.springframework.nativex.hint.NativeHint;
+import org.springframework.nativex.hint.ResourceHint;
+import org.springframework.nativex.hint.TypeHint;
+
+@SpringBootApplication
+@NativeHint(options = "--enable-url-protocols=https")
+@ResourceHint(patterns = "com/okta/commons/configcheck/configuration-validator", isBundle = true)
+@TypeHint(typeNames = {
+    "com.okta.spring.boot.oauth.OktaOpaqueTokenIntrospectConditional",
+    "com.okta.spring.boot.oauth.OktaOpaqueTokenIntrospectConditional$ClientIdCondition",
+    "com.okta.spring.boot.oauth.OktaOpaqueTokenIntrospectConditional$ClientSecretCondition",
+    "com.okta.spring.boot.oauth.OktaOpaqueTokenIntrospectConditional$IntrospectionUriCondition"
+}, access = AccessBits.ALL)
+public class DemoApplication {
+
+    public static void main(String[] args) {
+        SpringApplication.run(DemoApplication.class, args);
+    }
+}
+```
+
+We are working on providing a native hints dependency with this configuration. This will allow you to add a dependency to your build file without making code changes.
```

Comment:
I created a sample repo to show how this might be done: https://github.com/mraible/okta-native-hints

Can we add it to this repo and publish it along with other artifacts? It sure would be sweet if we could!

Reply:
@arvindkrishnakumar-okta ^^